model/src/channel/mod.rs

```diff
@@ -116,7 +118,12 @@ impl<'de> Visitor<'de> for GuildChannelVisitor {
             let key = match map.next_key() {
                 Ok(Some(key)) => key,
                 Ok(None) => break,
-                Err(_) => continue,
+                Err(_) => {
```

Comment:
Would it be worth putting this in a `trace` or `debug` log?

Reply:
This should probably be handled separately via https://github.com/twilight-rs/twilight/issues/44, otherwise since we only have a small number of custom deserializers as a percentage of all the models the logs won't be too useful.